jupyter/restore_raft_quorum_with_tablets.ipynb

```diff
@@ -0,0 +1,680 @@
+    "parallel_stop.run(stop_scylla_server_not_gently)\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "127443ea",
+   "metadata": {},
+   "source": [
+    "# GET CLUSTER STATUS\n",
+    "\n",
+    "This cell could be used to validate current status of cluster with `nodetool status`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "58689968",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "verification_node = nodes_by_region[alive_dc_region][0]\n",
+    "print(verification_node.run_nodetool(\"status\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de494749",
+   "metadata": {},
+   "source": [
+    "### Validate that raft quorum lost\n",
+    "Try to remove node dead node\n",
+    "Call read barrier.\n",
+    "\n",
+    "Both operations should failed with exception"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c479fbfb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "try:\n",
+    "    verification_node.run_nodetool(f\"removenode {dead_node_host_ids[0]} --ignore-dead-nodes {','.join(dead_node_host_ids[1:])}\")\n",
+    "except Exception as exc:\n",
+    "    print(f\"Expected to fail. Error: {exc}\")\n",
+    "\n",
+    "try:\n",
+    "    verification_node.raft.call_read_barrier()\n",
+    "except Exception as exc:\n",
+    "    print(f\"Expected to fail. Error: {exc}\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "571d6f73",
+   "metadata": {},
+   "source": [
+    "# Recovery procedure\n",
+    "Execute each step one by one\n",
+    "Use 'GET CLUSTER STATUS' cell to check cluster status between steps if need"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f632964f",
+   "metadata": {},
+   "source": [
+    "## Step 2.\n",
+    "Stop one of alive node in alive dc.\n",
+    "Assume that it is the latest node in list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1f1893f8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "stopped_alive_node = nodes_by_region[alive_dc_region][-1]\n",
+    "stopped_node_host_id = stopped_alive_node.host_id\n",
+    "stopped_alive_node.stop_scylla()\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a7c8a1ce",
+   "metadata": {},
+   "source": [
+    "## Step 4.\n",
+    "Choose alive nodes which will be used for cluster restore\n",
+    "Rolling restart of alive nodes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42acf42c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "base_alive_nodes = nodes_by_region[alive_dc_region][:-1]\n",
+    "cluster.restart_scylla(nodes=base_alive_nodes)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b4fc3063",
+   "metadata": {},
+   "source": [
+    "## Step 6. \n",
+    "Find recovery coordinator"
```

Comment:
We call it "recovery leader" everywhere. It would be good replace all "coordinator" occurrences with "leader" in this PR.